ocaml/src/albamgr_client.ml

```diff
@@ -651,7 +651,12 @@ let make_client buffer_pool (ccfg:Arakoon_client_config.t) ~tcp_keepalive =
   find_master' ~tls ~tcp_keepalive ccfg >>= function
   | MasterLookupResult.Found (m , ncfg) ->
      let open Arakoon_client_config in
-     let conn_info = Networking2.make_conn_info ncfg.ips ncfg.port tls_config in
+     let transport = Net_fd.TCP in
```

Comment:
Hmm, should this be considered a TODO?
Or is the idea here that if you want RDMA with arakoon then you'll have to use the preload lib anyway and it should all work fine?
Btw did you test the combination of preload + using rsockets towards the asds? (Just wondering if that might give issues)